src/Functions/castOrDefault.cpp

```diff
@@ -376,66 +376,475 @@ SELECT accurateCastOrDefault('abc', 'UInt32')
+    FunctionDocumentation::Description toInt256OrDefault_description = R"(
+Like [`toInt256`](#toint256), this function converts an input value to a value of type [Int256](../data-types/int-uint.md) but returns the default value in case of an error.
+If no `default` value is passed then `0` is returned in case of an error.
+    )";
+    FunctionDocumentation::Syntax toInt256OrDefault_syntax = "toInt256OrDefault(expr[, default])";
+    FunctionDocumentation::Arguments toInt256OrDefault_arguments = {
+        {"expr", "Expression returning a number or a string representation of a number.", {"String", "Int", "Float"}},
+        {"default", "Optional. The default value to return if parsing is unsuccessful.", {"Int256"}}
+    };
+    FunctionDocumentation::ReturnedValue toInt256OrDefault_returned_value = {"Value of type Int256 if successful, otherwise returns the default value if passed or 0 if not.", {"Int256"}};
+    FunctionDocumentation::Examples toInt256OrDefault_examples = {
+        {"Successful conversion", "SELECT toInt256OrDefault('-256', CAST('-1', 'Int256'))", "-256"},
+        {"Failed conversion", "SELECT toInt256OrDefault('abc', CAST('-1', 'Int256'))", "-1"}
+    };
+    FunctionDocumentation::Category toInt256OrDefault_category = FunctionDocumentation::Category::TypeConversion;
+    FunctionDocumentation::IntroducedIn toInt256OrDefault_introduced_in = {21, 11};
+    FunctionDocumentation toInt256OrDefault_documentation = {toInt256OrDefault_description, toInt256OrDefault_syntax, toInt256OrDefault_arguments, toInt256OrDefault_returned_value, toInt256OrDefault_examples, toInt256OrDefault_introduced_in, toInt256OrDefault_category};
+
     factory.registerFunction("toInt256OrDefault", [](ContextPtr context)
-        { return std::make_shared<FunctionCastOrDefaultTyped>(context, "toInt256OrDefault", std::make_shared<DataTypeInt256>()); });
+        { return std::make_shared<FunctionCastOrDefaultTyped>(context, "toInt256OrDefault", std::make_shared<DataTypeInt256>()); },
+        toInt256OrDefault_documentation);
+
+    FunctionDocumentation::Description toFloat32OrDefault_description = R"(
+Like [`toFloat32`](#tofloat32), this function converts an input value to a value of type [Float32](../data-types/float.md) but returns the default value in case of an error.
+If no `default` value is passed then `0` is returned in case of an error.
+    )";
+    FunctionDocumentation::Syntax toFloat32OrDefault_syntax = "toFloat32OrDefault(expr[, default])";
+    FunctionDocumentation::Arguments toFloat32OrDefault_arguments = {
+        {"expr", "Expression returning a number or a string representation of a number.", {"String", "Int", "Float"}},
+        {"default", "Optional. The default value to return if parsing is unsuccessful.", {"Float32"}}
+    };
+    FunctionDocumentation::ReturnedValue toFloat32OrDefault_returned_value = {"Value of type Float32 if successful, otherwise returns the default value if passed or 0 if not.", {"Float32"}};
```

Comment:
```suggestion
    FunctionDocumentation::ReturnedValue toFloat32OrDefault_returned_value = {"Returns a value of type Float32 if successful, otherwise returns the default value if passed or 0 if not.", {"Float32"}};
```